include/ignition/gazebo/EventManager.hh

```diff
@@ -140,9 +140,11 @@ namespace ignition
                  }
                };
 
+      /// This was originally a unique_ptr, but pybind11 was failing because it
+      /// was not able to copy the class
       /// \brief Container of used signals.
       private: std::unordered_map<TypeInfoRef,
-                                  std::unique_ptr<ignition::common::Event>,
+                                  std::shared_ptr<ignition::common::Event>,
```

Comment:
Can we revert this change like you did for the ECM? Can the same logic be applied here? Otherwise we'll need to retarget the PR to Garden because this breaks ABI.